Source/WebCore/platform/mediarecorder/MediaRecorderPrivateGStreamer.cpp

```diff
@@ -170,16 +201,16 @@ void MediaRecorderPrivateBackend::stopRecording(CompletionHandler<void()>&& comp
 
 void MediaRecorderPrivateBackend::fetchData(MediaRecorderPrivate::FetchDataCallback&& completionHandler)
 {
-    callOnMainThread([this, weakThis = ThreadSafeWeakPtr { *this }, completionHandler = WTFMove(completionHandler)]() mutable {
+    callOnMainThread([this, weakThis = ThreadSafeWeakPtr { *this }, completionHandler = WTFMove(completionHandler), mimeType = this->mimeType()]() mutable {
         auto protectedThis = weakThis.get();
         if (!protectedThis) {
-            completionHandler(nullptr, mimeType(), 0);
+            completionHandler(nullptr, mimeType, 0);
             return;
         }
         Locker locker { m_dataLock };
-        GST_DEBUG_OBJECT(m_transcoder.get(), "Transfering %zu encoded bytes", m_data.size());
+        GST_DEBUG_OBJECT(m_transcoder.get(), "Transfering %zu encoded bytes, mimeType: %s", m_data.size(), mimeType.ascii().data());
         auto buffer = m_data.take();
-        completionHandler(WTFMove(buffer), mimeType(), m_position);
+        completionHandler(WTFMove(buffer), mimeType, m_position);
     });
```

Comment:
Why?

Reply:
Why what?

Reply:
I answered myself. You need to capture the mime in case the protected this is gone...